miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -565,24 +558,11 @@ export.authenticate_procedure
 
     # get procedure info (PROC_ELEMENTS, storage_offset) from memory stored at index
     exec.get_procedure_info
-    # In production:
     # => [PROC_ELEMENTS, storage_offset, PROC_ROOT]
```

Comment:
I would change the return naming here from `PROC_ELEMENTS` to `PROC_ROOT` and in the above comment.

Reply:
Done

Reply:
Done!